package.json

```diff
@@ -19,8 +19,8 @@
     "node": ">=18.15.0"
```

Comment:
Let's update this repo to support the new Node support policy https://auro.alaskaair.com/node-support